server/test/Main.hs

```diff
@@ -61,25 +107,26 @@ serverSpec = do
   describe "Api.Handlers.hashScript" hashScriptSpec
   describe "Api.Handlers.blake2bHash" blake2bHashSpec
   describe "Api.Handlers.hashData" hashDataSpec
+  describe "Ogmios.Parser " testParser
 
 applyArgsSpec :: Spec
 applyArgsSpec = around withTestApp $ do
   clientEnv <- setupClientEnv
 
   context "POST apply-args" $ do
-    it "returns the same script when called without args" $ \port -> do
+    it "pures the same script when called without args" $ \port -> do
```

Comment:
A nice purescript/haskell joke here, 

I hope it was done intentionally and is not a result of bad string replace.

Reply:
gotcha, i forgot `return` is a valid English word.